src/appleseed-max-impl/appleseeddisneymtl/appleseeddisneymtl.cpp

```diff
@@ -855,3 +861,8 @@ HINSTANCE AppleseedDisneyMtlClassDesc::HInstance()
 {
     return g_module;
 }
+
+bool AppleseedDisneyMtlClassDesc::IsCompatibleWithRenderer(ClassDesc& renderer_class_desc)
+{
+    return (renderer_class_desc.ClassID() == AppleseedRenderer::get_class_id()) ? true : false;
```

Comment:
You could simplify even further: the `? true : false` part is not needed since the expression is already of type `bool`. Let's leave it like that for now. I might make that simplification later.